docs/customizing-toolbar.md

```diff
@@ -20,6 +20,7 @@ function App() {
       markdown="Hello world"
       plugins={[
         toolbarPlugin({
+          toolbarClassName: 'my-classname',
```

Comment:
Why not just `className`, or , maybe`rootElementClassName`?

Reply:
@petyosi to keep naming consistent across the toolbar component. There is `toolbarContent`, so `toolbarClassName` points user directly to the `Root` element (so the wrapper around the content itself).

Reply:
`toolbarClassName` gives user pretty good understanding where this class will be applied (also good for the future, when there might be more classess for different elements).